### PR TITLE
Re-enabled function tests

### PR DESCRIPTION
**Pull request by imalberto**
- Re-enable yui_module test suite
- Re-enable configtest suite
- Delete html5app test suite

### CODE REVIEW

File: tests/run.js [hunk 1]
```diff
@@ -372,22 +372,6 @@ function runFuncAppTests(cmd, callback) {
     //
     // TODO: Fix those tests
     skips = [
```
Comment (imalberto) [inline tests/run.js:374]: No more tests to skip.

File: tests/func/common/html5apptest_descriptor.json [hunk 2]
```diff
@@ -1,444 +0,0 @@
-[
```
Comment (imalberto) [inline tests/func/common/html5apptest_descriptor.json:1]: `html5app` tests suite removed - no longer supported.

File: package.json [hunk 3]
```diff
@@ -47,6 +47,7 @@
         "benchmark": "1.x",
         "commander": "1.0.1",
         "express": "3.4.1",
+        "minimist": "~0.0.5",
```
Comment (imalberto) [inline package.json:50]: Used by examples app to parse command line arguments.

**Comment by caridy** [comment]
+1

**Pull request merged**